src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import React, { useEffect } from 'react';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  useEffect(() => {
+    if (!tabs.find(tab => tab.id === activeTabId)) {
+      onTabSelected(tabs[0].id);
+    }
+  }, [activeTabId, onTabSelected, tabs]);
+
+  return (
+    <>
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              data-cy="Tab"
+              className={tab.id === activeTab.id ? 'is-active' : ''}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={e => {
+                  e.preventDefault();
+                  if (tab.id !== activeTab.id) {
+                    onTabSelected(tab.id);
+                  }
+                }}
```

Comment:
Pls move the function inside `onClick` handler to a separate const and use as a callback here

Reply:
understood